test/spec/app/components/footer/footer.component.spec.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+describe("component test", function () {
+
+    beforeEach(module("FoodVotingApp"));
+
+    describe("footer component", function () {
+
+        var $componentController;
+
+        beforeEach(inject(function (_$componentController_) {
+            $componentController = _$componentController_;
+        }));
+
+        it("should test on footer component", function () {
+
+            expect($componentController).toBeDefined();
+
+        });
+
+    });
+
+});
```

Comment:
Add test for `getAppName` to check return value or check value of `appName` after init.